config/elasticache/config.go

```diff
@@ -11,15 +11,15 @@ import (
 // Configure adds configurations for elasticache group.
 func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("aws_elasticache_cluster", func(r *config.Resource) {
-		r.References = config.References{
-			"parameter_group_name": config.Reference{
-				Type: "ParameterGroup",
-			},
-			"subnet_group_name": config.Reference{
-				Type: "SubnetGroup",
-			},
+		r.References["parameter_group_name"] = config.Reference{
+			TerraformName: "aws_elasticache_parameter_group",
 		}
-		r.UseAsync = true
+		r.References["subnet_group_nameZ"] = config.Reference{
```

Comment:
```suggestion
		r.References["subnet_group_name"] = config.Reference{
```

Reply:
Thanks for catching this :)
Looks like the reference is already auto-injected and does not need manual configuration. Removed the `subnet_group_name` reference configuration.